static/app/views/replays/detail/ai/chapterList.tsx

```diff
@@ -357,11 +356,12 @@ const Chapter = styled('summary')`
 `;
 
 const ChapterTitle = styled('div')`
-  display: flex;
+  display: grid;
+  grid-template-columns: 4fr 2fr;
```

Comment:
What was wrong with having it has flex? - this makes it so that grid columns are a fixed ratio size where as previously the "title" el could take up all of the space

Reply:
alternatively with grid, you could size it as `auto max-content` or `auto auto`